### ISSUE TITLE
Fix pagination for docx exports

### ISSUE THREAD

**Issue by Melkiades** [insightsengineering/rtables#939]
Issues:
1. Extracted widths from dim.flextable() but also dim_pretty() are inexact (they should be in inches but they do not seem to match) -> Manual setting of widths depends on total_page_width that needs to be changed following this uncertainties.
2. Similarly lpp estimation is impossible with faulty dim estimation from flextable. When inexact it also appends tables on the same page, to an inconvient degree. A possible fix could be to **impose a different page each call to export_as_docx in the loop** but I cannot find a way to do it. Still would be hard to determine the right lpp.
3. `cpp` is not considered atm for 1.
4. Possible issues with titles and footers positions (are part of the table now. We might want them only before and after the table.

point 2 (the new page imposition) needs to be solved for `page_by` functionality.

I believe that point 2 calculation may need to go into docx engine. We may want at the end use JL workaround of collapsing cells to avoid splitting them. 

In summary: consider fixing pagination sizes with reverting to only word pagination system. Current `dim` gives inexact dimensions
